kubernetes_state_core/metadata.csv

```diff
@@ -139,3 +139,4 @@ kubernetes_state.service.type,gauge,,,,Service types. Tags:`kube_namespace` `kub
 kubernetes_state.ingress.count,gauge,,,,Number of ingresses. Tags:`kube_namespace`.,0,kubernetes_state_core,k8s_state.ingress.count,
 kubernetes_state.ingress.path,gauge,,,,Information about the ingress path. Tags:`kube_namespace` `kube_ingress_path` `kube_ingress` `kube_service` `kube_service_port` `kube_ingress_host` .,0,kubernetes_state_core,k8s_state.ingress.path,
 kubernetes_state.crd.count,gauge,,,,Number of crds. Tags:`kube_namespace`.,0,kubernetes_state_core,k8s_state.crd.count,
+kubernetes_state.pod.tolerations,gauge,,,,Information about the pod tolerations,0,kubernetes_state_core,k8s_state.pod.tolerations,
```

Comment:
There are some kubernetes_state.pod.* metrics defined above. It might be better to place this one above so all of them are together.

Reply:
> There are some kubernetes_state.pod.* metrics defined above. It might be better to place this one above so all of them are together.

Up to you but we could merge the PR and hold off syncing the tile too until 7.48.0 is out

Reply:
You are right, I moved it up to a more suitable location

thanks